www/src/Locales/en/AddonsConfig.jsx

```diff
@@ -56,6 +56,19 @@ export default {
 	'dual-directional-input-dpad-mode-label': 'Dual D-Pad Mode',
 	'dual-directional-input-combine-mode-label': 'Combination Mode',
 	'dual-directional-input-four-way-joystick-mode-label': 'Dual Directional 4-Way Joystick Mode',
+	'AddonsConfig:tilt-header-text': 'Tilt Input',
+	'AddonsConfig:tilt-1-pin-label': 'Tilt 1 Pin',
+	'AddonsConfig:tilt-2-pin-label': 'Tilt 2 Pin',
+	'AddonsConfig:tilt-left-analog-up-pin-label': 'Tilt Left Analog Up Pin',
+	'AddonsConfig:tilt-left-analog-down-pin-label': 'Tilt Left Analog Down Pin',
+	'AddonsConfig:tilt-left-analog-left-pin-label': 'Tilt Left Analog Left Pin',
+	'AddonsConfig:tilt-left-analog-right-pin-label': 'Tilt Left Analog Right Pin',
+	'AddonsConfig:tilt-right-analog-up-pin-label': 'Tilt Right Analog Up Pin',
+	'AddonsConfig:tilt-right-analog-down-pin-label': 'Tilt Right Analog Down Pin',
+	'AddonsConfig:tilt-right-analog-left-pin-label': 'Tilt Right Analog Left Pin',
+	'AddonsConfig:tilt-right-analog-right-pin-label': 'Tilt Right Analog Right Pin',
+	'AddonsConfig:tilt-socd-mode-label': 'Tilt SOCD Mode',
+	'AddonsConfig:enabled-label': 'Enabled',
```

Comment:
This can be removed and you can use `Common:switch-enabled` as the key.

Reply:
Thank you for your review. I have corrected both errors. Could you please review this code again?

Reply:
Great!